src/Data/Fin/Subset/Properties.agda

```diff
@@ -220,9 +220,9 @@ x∉⁅y⁆⇒x≢y x∉⁅x⁆ refl = x∉⁅x⁆ (x∈⁅x⁆ _)
 infix 4 _⊆?_
 _⊆?_ : B.Decidable {A = Subset n} _⊆_
 []          ⊆? []          = yes id
-outside ∷ p ⊆?       y ∷ q = Dec.map out⊆-⇔ (p ⊆? q)
+outside ∷ p ⊆?       y ∷ q = map′ out⊆ drop-∷-⊆ (p ⊆? q)
```

Comment:
Is there a good reason for all of these changes from the higher-level `map` interface to the lower-level `map′ ` interface? Does it reduce the dependencies somehow? Otherwise I'm not quite sure what the value is? It makes some of the proofs much longer, e.g. `anySubset?` below.

Reply:
Yes, it does remove the dependency on `Relation.Nullary.Decidable` in favour of the already-being-imported `Relation.Nullary.Decidable.Core`... and when reaidng the proofs, I was realising that within a single module (in each case), an `Equivalence`/`Inverse` bundle was being created only to then be destructed by `Dec.map`, when the pieces of each bundle were already defined. But indeed, perhaps it is too clumsy a refactoring?